infinity/test_infinity.py

```diff
@@ -45,11 +46,15 @@ def get_gpu_count():
 @pytest.mark.parametrize("new_connection", [True, False])
 @pytest.mark.parametrize("local", [True, False])
 def test_basic_read_write_cache(server, dtype, new_connection, local):
-    conn = infinity.InfinityConnection()
+    config = ClientConfig()
+    config.service_port = 22345
+    config.host_addr = "10.192.24.218"
```

Comment:
??this is unit test.  only '127.0.0.1' is allowed

Reply:
could use a global Client/Server config.

Reply:
fixed